### ISSUE TITLE
Why is it loaded, even when I announced it in user settings?

### ISSUE THREAD

**Issue by baxebu** [pichillilorenzo/JavaScript-Completions#44]
Hello! Why is it loaded, even when I announced it in user settings?
![screenshot](https://user-images.githubusercontent.com/33215968/32158663-78e386a4-bd7c-11e7-80aa-4c1651013abc.jpg)

**Comment by pichillilorenzo** [comment]
Hi, the one you see is installed for internal use.

**Comment by baxebu** [comment]
Yes, but I want to use the latest version of node? And here it downloads the old version of node.
